templates/hoover.nomad

```diff
@@ -196,6 +230,17 @@ job "hoover" {
               {{- end }}
             {{- end }}
 
+            {{- if service "tika" }}
```

Comment:
This `if` has no effect, since `with index . 0` on an empty sequence will render to an empty string.